ucsb-cs156-f24/proj-happycows-f24-10
# Add Course database table

## Overview

In this PR, I add a database table and corresponding api endpoint to store courses (without linking the student table yet) as well as tests for the CourseController.

## Feedback Request (Optional)

Please give suggestions on the following:
- Is there any other information we should have for every course?
- Should we include start and end dates (and should those attempt to validate with commons start and end dates)

## Future Possibilities (Optional)

This is part of epic #6. The two immediate ways to build on this commit are to add a frontend interface and to connect it to the Students table once both are merged.

## Validation (Optional)

1. Download this branch.
2. Check the swagger locally or on dokku for the courses api endpoint

## Tests


## Linked Issues

Closes #11
